x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_text_property.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DynamicStyleProperty } from './dynamic_style_property';
+import { getComputedFieldName } from '../style_util';
+
+export class DynamicTextProperty extends DynamicStyleProperty {
+
+  syncTextFieldWithMb(mbLayerId, mbMap) {
+    const targetName = getComputedFieldName(this._styleName, this._options.field.name);
```

Comment:
Is it possible to create a helper function to limit the decimal precision of the label value when necessary. For example, the Metrics visualization appears to limit the precision to up to three decimal places. 

<img width="1047" alt="Screen Shot 2019-12-13 at 9 25 25 AM" src="https://user-images.githubusercontent.com/1638483/70822052-7a725c80-1d91-11ea-8b94-fe24d72a2137.png">

Reply:
With regular fields, we can always recommend scripted fields to format the label to accommodate user requirements but with aggregated fields, the issue you mention is super common. 

For starters, we may want to just establish simple and predictable rules like the one you mention, but eventually, users will want to have more control of the formatting like adding currency symbols and other prefix/suffix strings. This is super exciting!!

Reply:
@nickpeihl I have merged a commit that uses the index-pattern field formatter to format the label. Numbers default to a number formatter that rounds to 3 decimal places but users can now update the field formatter config for the field to customize as needed and the display will be consistent across kibana applications